src/Server/HTTPHandlerFactory.cpp

```diff
@@ -184,13 +199,8 @@ void addCommonDefaultHandlersFactory(HTTPRequestHandlerFactoryMain & factory, IS
     root_handler->allowGetAndHeadRequest();
     factory.addHandler(root_handler);
 
-    auto ping_handler = std::make_shared<HandlingRuleHTTPHandlerFactory<StaticRequestHandler>>(server, ping_response_expression);
```

Comment:
@ye11ow does neutron depend on `http://localhost:8123/ping` ? We have `http://localhost:8123/proton/ping` before and the latter returns a json response.

Reply:
Neutron uses `http://localhost:8123/proton/ping` (we don't use `http://localhost:8123/ping`).
Will this be impacted?

Reply:
no. i am removing `http://localhost:8123/ping`